test/test.js

```diff
@@ -4,22 +4,27 @@ var path = require('path');
 var babel = require('babel-core');
 var filterImports = require('../index');
 
-function testFixture(name, options) {
+function testFixtureWithPlugins(name, plugins) {
   it(name, function () {
     var fixturePath = path.resolve(__dirname, 'fixtures', name, 'fixture.js');
     var expectedPath = path.resolve(__dirname, 'fixtures', name, 'expected.js');
 
     var expected = fs.readFileSync(expectedPath).toString();
     var result = babel.transformFileSync(fixturePath, {
-      blacklist: ['strict'],
-      modules: 'commonStrict',
-      plugins: [filterImports(options)]
+      babelrc: false,
+      plugins: plugins,
     });
 
-    assert.strictEqual(result.code, expected);
+    assert.strictEqual(result.code, expected.trim());
   });
 }
 
+function testFixture(name, options) {
+  testFixtureWithPlugins(name, [
+    [filterImports, options],
```

Comment:
@mmun I'm wondering if we should wrap the imports in an options object

```js
[filterImports, {
  imports: { assert: ['default'], cloud: ['default'] },
}]

// instead of

[filterImports, { assert: ['default'], cloud: ['default'] }]
```